index.html

```diff
@@ -32,60 +32,67 @@
       </ul>
     </nav>
 
-    <main class="wrapper">
-      <section class="left">
-        <h2 class="hide">Section Header</h2>
-        <article class="border-box-1, content-box-1">
-          <h2 id="a1">Come to Academy Archery and see why we are the best Archery place in town!</h2>
+    <main class="wrapper grid">
+        <section class="row">
+          <h2 class="hide">Section Header</h2>
+          <article class="col span_8_of_12">

```

Comment:
Using the 12-column grid layout here works perfectly well, but remember to use CSS Grid somewhere in the project for full credit. 
You could do that in a particular section, but if you want a challenge, see how you could achieve the same grid layout after deleting all of these `.col` `.row` and span classes from the CSS and rely only on CSS Grid.